### PR TITLE
Change text for File

### PR DESCRIPTION
**Pull request by ZerooCool**
Become Folders and Files

### CODE REVIEW

**Comment by PHLAK** [comment]
What about `Name` instead?

**Comment by ZerooCool** [comment]
Sorry I did not understand.

Here, I simply proposed to improve a static text, because, there are folders and files, in the named colone. So you have to put the 2 possibilities, in the master version, by default.

Free to the user to modify, but, both are necessary by default, since the tool makes it possible to display the files and the files.

File: resources/themes/bootstrap/index.php [hunk 1]
```diff
@@ -86,7 +86,7 @@
 
             <div id="directory-list-header">
                 <div class="row">
-                    <div class="col-md-7 col-sm-6 col-xs-10">File</div>
+                    <div class="col-md-7 col-sm-6 col-xs-10">Folders and Files</div>
```
Comment (PHLAK) [inline resources/themes/bootstrap/index.php]:
```suggestion
                    <div class="col-md-7 col-sm-6 col-xs-10">Name</div>
```

**Comment by PHLAK** [comment]
> Sorry I did not understand.

Can we change the `Folders and Files` title to `Name` instead?

**Comment by PHLAK** [comment]
Good news, Directory Lister 3 is now in beta! View the [release announcement](https://medium.com/chriskankiewicz/directory-lister-3-beta-is-here-d821236ac911) for more info.

This means all pull requests opened against the 2.X (currently `master`) branch will now be closed. We encourage you to to download and try the Directory Lister 3 beta and open a new PR if necessary.

If you'd like to join the discussion regarding the ongoing development of Directory Lister 3 you may do so in our [community forum](https://spectrum.chat/directory-lister).

**Comment by ZerooCool** [comment]
Name should be good, yes. Thank you.

**Pull request closed without merging**